### ISSUE TITLE
[a11y] Unable to submit form with 'Enter' key

### ISSUE THREAD

**Issue by lachieh** [gautamkrishnar/nothing-private#130]
**Describe the bug**
When using the main form, it is not possible to submit the form using the enter key, while the text input is focused. This is not really an issue, but it is better for accessibility (visual and motor ability) to use semantic elements and their standard behavior. Since this is actually submitting data, a form element is more suited.

**To Reproduce**
1. Go to main page
2. Click on the only text input
3. Press enter key
4. Nothing happens

**Expected behavior**
As above, except:
4. Form submits

**Context:**
All browsers
